packages/node_modules/@node-red/nodes/core/common/25-status.js

```diff
@@ -21,6 +21,15 @@ module.exports = function(RED) {
         RED.nodes.createNode(this,n);
         var node = this;
         this.scope = n.scope;
+
+        // auto-filter out any directly connected nodes to avoid simple loopback
+        const w = this.wires.flat().toString();
```

Comment:
```suggestion
        const w = this.wires.flat();
```
```suggestion
        const w = this.wires.flat();
```

Safer to use Array.includes rather than String.includes - avoid partial incorrect matches.